template.tpl

```diff
@@ -489,9 +496,6 @@ function mapEvent(eventData, data) {
     mappedData = overrideDataIfNeeded(data, mappedData);
     mappedData = cleanupData(mappedData);
 
-    logToConsole('Event raw data: ', eventData);
-    logToConsole('Snap mapped data: ', mappedData);
```

Comment:
These 2 were pretty useful 😅 . Should we add this into the internal dev document? And remove them for the external codes?

Reply:
Sure